pkg/artifactory/resource_artifactory_local_repository_test.go

```diff
@@ -107,6 +107,11 @@ func resourceLocalRepositoryCheckDestroy(id string) func(*terraform.State) error
 
 		_, resp, err := client.V1.Repositories.GetLocal(context.Background(), rs.Primary.ID)
 
+		if resp == nil {
```

Comment:
Check `err` first. If you check error first you probably never need to check if `resp` is `nil`.